README.md

```diff
@@ -1,3 +1,15 @@
+The Portals proposal is not currently being actively pursued. 
+
+Many of the capabilities that Portals would have enabled are being explored in other proposals:
+* [navigation speculation](https://github.com/WICG/nav-speculation) (including speculation rules, prefetch, and prerendering)
```

Comment:
nit: should we link to https://wicg.github.io/nav-speculation/ for consistency with the fenced frame link below?

LGTM otherwise!

Reply:
That page is mostly an index, and I think the README.md gives a better overview of the work.